common/app/conf/switches/CommercialSwitches.scala

```diff
@@ -357,6 +357,16 @@ trait CommercialSwitches {
     sellByDate = never,
     exposeClientSide = true
   )
+
+  val facebookTrackingPixel: Switch = Switch(
+    group = Commercial,
+    name = "facebook-tracking-pixel",
+    description = "Facebook's PageView tracking to improve ad targeting",
+    owners = group(Commercial),
+    safeState = Off,
```

Comment:
Shouldn't "safe state" be 'on' as it's the current situation?

Reply:
I was kind of comparing it to the ones around it which also default to Off. It's a good point though.

Reply:
The off state looks right in this case because by default features should be actively switched on.  So the only 'on' by default switches are where they're actually disabling something, generally.